third_party/docfx-doclet-143274/src/main/java/com/microsoft/util/ElementUtil.java

```diff
@@ -15,34 +17,54 @@
 
 public class ElementUtil {
 
+    private static final int INITIAL_CAPACITY = 1000000;

```

Comment:
Set large initial map capacity to *try* and reduce the number of copy/ increases needed.